### PR TITLE
Bug fix for jazz_cognito-admin-authorizer

### PR DESCRIPTION
**Pull request by supritat**
### Requirements

* Bug fix for jazz_cognito-admin-authorizer and jazz_create-serverless-service services.

### Description of the Change

* jazz_cognito-admin-authorizer: fixed constructor error that as arrow functions don't have this, they cannot be used as constructor.
* jazz_create-serverless-service: Passing authToken param to updateAclPolicy() and fix for error handling.

### Benefits

* jazz_cognito-admin-authorizer successfully validates the for admin authorization.
* jazz_create-serverless-service successfully send request to ACL service to update the policies for user created service.

### Possible Drawbacks

NA

### Applicable Issues

NA

### CODE REVIEW

File: core/jazz_cognito-admin-authorizer/index.js [hunk 1]
```diff
@@ -402,7 +402,7 @@ module.exports.handler = (event, context, cb) => {
        *
        * @method allowAllMethods
        */
-      allowAllMethods: () => {
+      allowAllMethods: function () {
```
Comment (svidhani) [inline core/jazz_cognito-admin-authorizer/index.js]: why?

File: core/jazz_cognito-admin-authorizer/index.js [hunk 2]
```diff
@@ -411,7 +411,7 @@ module.exports.handler = (event, context, cb) => {
        *
        * @method denyAllMethods
        */
-      denyAllMethods: () => {
+      denyAllMethods: function () {
```
Comment (svidhani) [inline core/jazz_cognito-admin-authorizer/index.js]: why?

Comment (supritat) [reply core/jazz_cognito-admin-authorizer/index.js]: It was throwing TypeError with arrow function. May be it is due to the context attached to "this".

Comment (supritat) [reply core/jazz_cognito-admin-authorizer/index.js]: It was throwing TypeError with arrow function. May be it is due to the context attached to "this".

File: core/jazz_cognito-admin-authorizer/index.js (hunk 1 above)
Comment (supritat) [reply core/jazz_cognito-admin-authorizer/index.js]: It was throwing TypeError with arrow function. May be it is due to the context attached to "this".

**Pull request merged**